### PR TITLE
docs: add initial contribution guidelines

### PR DESCRIPTION
**Pull request by emmenko**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Please _read_ before contributing to Test Data in order to get familiar with the guidelines for contributing to the project.
+
+## Core Ideas
+
+Writing tests often times requires to provide meaningful test data. But how can we do that consistently and how can we share this?
+
+We want to provide a shared implementation to define test data across applications that adheres to the following patterns:
+
+* Test data generation should not be provide inline in each test but be defined in a central and reusable place.
```
Comment (adnasa) [inline CONTRIBUTING.md:11]:
```suggestion
* Test data generation should not be provided inline in each test, but be defined in a central and reusable place.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* Test data should be randomly generated but still be as representative of real production data as possible
+* Test data generation should be customizable for different scenarios and not altered in the tests themselves.
+
+## Submitting a Pull Request
+
+Good pull requests, such as patches, improvements, and new features, are a fantastic help. They should remain focused in scope and avoid containing unrelated commits.
```
Comment (adnasa) [inline CONTRIBUTING.md:17]: keep it plural in this case 
```suggestion
Good pull requests, such as patches, improvements, and new features, are fantastic help. They should remain focused in scope and avoid containing unrelated commits.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Once it's done, you can run `yarn test` to test your changes.
+
+## Adding changesets
+
+commercetools test-data uses [changesets](https://github.com/atlassian/changesets) to do versioning and creating changelogs.
+
+As a contributor you need to add a changeset by running `yarn changeset`.
+The command will prompt to select the packages that should be bumped, their associated semver bump types and some markdown which will be inserted into the changelogs.
+
+When opening a Pull Request, a `changeset-bot` checks that the Pull Request contains a changeset. A changeset is **NOT required**, as things like documentation or other changes in the repository itself generally don't need a changeset.
```
Comment (adnasa) [inline CONTRIBUTING.md:42]: uppercase `not` is not required (no pun intended)

```suggestion
When opening a Pull Request, a `changeset-bot` checks that the Pull Request contains a changeset. A changeset is **not required**, as things like documentation or other changes in the repository itself generally don't need a changeset.
```

**Review by adnasa — APPROVED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## Releasing packages
+
+commercetools application-kit uses [changesets](https://github.com/atlassian/changesets) to do versioning and publishing a release.
```
Comment (tdeekens) [inline CONTRIBUTING.md:46]:
```suggestion
commercetools test-data uses [changesets](https://github.com/atlassian/changesets) to do versioning and publishing a release.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+A [Changesets release GitHub Action](https://github.com/changesets/action) opens a `Version Packages` Pull Request whenever there are some changesets that have not been released yet.
+
+When the `Version Packages` Pull Request gets merged, the Changesets release GitHub Action will automatically trigger the release.
+
+## Canary releases
+
+On `master` branch, we automatically publish **canary** releases from CI to the `canary` distribution channel, _after_ the build runs successfully.
```
Comment (tdeekens) [inline CONTRIBUTING.md:54]:
```suggestion
On `main` branch, we automatically publish **canary** releases from CI to the `canary` distribution channel, _after_ the build runs successfully.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+Canary releases are useful to test early changes that should not be released yet to `next` or `latest`. They are automatically triggered and released after a Pull Request merged into `master`, unless the commit message contains `[skip publish]`.
```
Comment (tdeekens) [inline CONTRIBUTING.md:56]:
```suggestion
Canary releases are useful to test early changes that should not be released yet to `next` or `latest`. They are automatically triggered and released after a Pull Request merged into `main`, unless the commit message contains `[skip publish]`.
```

File: CONTRIBUTING.md (hunk 5 above)
Comment (emmenko) [reply CONTRIBUTING.md:54]: Thanks

**Review by tdeekens — APPROVED** [review]

**Pull request merged**